### DISCUSSION TITLE
Raw data for CitationFull datasets

### DISCUSSION THREAD

**Discussion by wwbrannon** [pyg-team/pytorch_geometric#5953] [Q&A] [answered]
Hi,

I'm looking to use the Cora/CiteSeer/Pubmed datasets from [CitationFull](https://pytorch-geometric.readthedocs.io/en/latest/modules/datasets.html#torch_geometric.datasets.CitationFull), but with a twist: I need to use the raw paper text, which means I need to know which papers are in the datasets. Some webscraping to get the text is fine, but it's surprisingly hard to find which papers the datasets contain, especially for CiteSeer. 

Does PyG include this information somewhere? Something like a numpy array of URLs for the papers? (Or do you know where to find it?) I haven't been able to find it in the docs.

Thanks

**Comment by rusty1s** [comment, marked as answer]
Sadly no :( You might have some success asking in https://github.com/abojchevski/graph2gauss for further information. @abojchevski

**Reply by wwbrannon** [reply]
thanks!
